src/package.json

```diff
@@ -43,6 +43,12 @@
     "test:infra": "./node_modules/.bin/jest --testMatch '**/infra/?(*.)test.ts' ",
     "api": "JEST_JUNIT_OUTPUT_NAME=api-report.xml ./node_modules/.bin/jest --runInBand --testPathPattern=tests/api/",
     "test:api": "npm run compile && npm run api",
+    "yoti": "JEST_JUNIT_OUTPUT_NAME=api-report.xml ./node_modules/.bin/jest --runInBand --testPathPattern=tests/YotiStub.test.ts",
```

Comment:
Does this testPathPattern match any of our test files?

Reply:
It does, but we have 2x YotiStub test files. 

1 in _src/tests/YotiStub.test.ts_ 

And another in _src/tests/api/YotiStub.test.ts_ 

Please clean up